forms/src/interfaces/iQuestion.ts

```diff
@@ -0,0 +1,17 @@
+import { iAnswer } from "./iAnswer";
+
+export interface iQuestion {
+  id?: string;
+  questionName: string;
+  questionType: QuestionType;
+  answers?: Array<iAnswer>;
+  selectedAnswers?: Array<iAnswer>;
+}
+export enum QuestionType {
+  shortAnswer = "shortAnswer",
+  longAnswer = "longAnswer",
+  radio = "radio",
+  checkbox = "checkbox",
+  select = "select",
+  title = "title",
+}
```

Comment:
String enums should be uppercased.

```suggestion
export enum QuestionType {
  shortAnswer = "SHORT_ANSWER",
  longAnswer = "LONG_ANSWER",
  radio = "RADIO",
  checkbox = "CHECKBOX",
  select = "SELECT",
  title = "TITLE",
}
```